src/app/course-item/course-item.component.html

```diff
@@ -0,0 +1,14 @@
+<header class="course__header">
+    <h2>{{course.name}}</h2>
+    <section class="course__header-info">
+        <p>{{course.duration}} minutes </p>
+        <p>&nbsp;{{course.date.getDate()}}.{{course.date.getMonth()}}.{{course.date.getFullYear()}}</p>
```

Comment:
Please remove this monstrosity with an elegant output of date through the `date` filter